Automattic/jetpack
# Forms: Dashboard finetuning

## Proposed changes:

* Several improvements as listed below:

- [x] Update View switcher dropdown name & description
- [x] Handle multi-line texts in the response details panel
- [x] Show `-` instead of an empty field in the response details panel.
- [x] Add links to the source forms in the response list table
- [x] Avoid showing emails twice in the response details panel
- [x] Show pagination only after 2 or more pages
- [x] Reset scroll state when switching between responses

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Enable the jetpack forms package and the new dashboard before testing with:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

- Go to `/wp-admin/admin.php?page=jetpack-forms`
- Check if the items listed above work as expected now
